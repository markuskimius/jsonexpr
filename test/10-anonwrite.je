FOR(i=0, i<2, i++,
    a = [1,2,3];

    PRINT(a);

    a[1] = "x";
);

PRINT(a);
