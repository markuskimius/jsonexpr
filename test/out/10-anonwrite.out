[ 1, 2, 3 ]
[ 1, 2, 3 ]
[ 1, "x", 3 ]
